scripts/utils.sh

```diff
@@ -8,25 +10,37 @@ function url_reachable() {
     return $?
 }
 
-
 function spawn_port_forwarding_command() {
-  kill_portforwarding_loop $1 $2 $3
-  run_in_background "scripts/port_forwarding_loop.sh $1 $2 $3"
+  service_name=$1
+  external_port=$2
+
+  cat << EOF > /etc/xinetd.d/${service_name}
+service ${service_name}
+{
+  flags		= IPv4
+  bind		= $(get_main_ip)
+  type		= UNLISTED
+  socket_type	= stream
+  protocol	= tcp
+  user		= root
+  wait		= no
+  redirect	= $(minikube ip) $(kubectl --kubeconfig=${KUBECONFIG} get svc/${service_name} -n assisted-installer -o=jsonpath='{.spec.ports[0].nodePort}')
+  port		= ${external_port}
+  only_from	= 0.0.0.0/0
+  per_source	= UNLIMITED
 }
+EOF
 
-function run_in_background() {
-  bash -c "nohup $1  >/dev/null 2>&1 &"
+  systemctl start xinetd
+  systemctl reload xinetd
 }
 
-
-function kill_portforwarding_loop() {
-  kill -9 $(ps aux | grep "port_forwarding_loop.sh $1 $2 $3" | grep -v grep | awk '{print $2}') || true
-  kill -9 $(ps aux | grep "kubectl --kubeconfig=${KUBECONFIG} port-forward" | grep $3 | grep -v grep | awk '{print $2}') || true
+function run_in_background() {
+  bash -c "nohup $1  >/dev/null 2>&1 &"
 }
 
 function kill_all_port_forwardings() {
-  kill -9 $(ps aux | grep "port_forwarding_loop" | grep -v grep | awk '{print $2}') || true
-  kill -9 $(ps aux | grep "port-forward" | grep -v grep | awk '{print $2}') || true
+  systemctl stop xinetd
```

Comment:
not sure regarding stop of the service, don't u think will be cleaner just to delete files and reload? To say the truth not sure we need kill_all_port_forwardings at all

Reply:
I can do it as well.
You are correct that in case somebody else uses xinet service I'd interrupt it.
But as the name suggests - I stopped **all** - like before **all** the port-forwarding were killed without paying attention if its related to our namespace or not.

Reply:
yap, but i don't really think we even need to kill them now. You can decide what is better

Reply:
For now I don't want to change API/behavior. Let's see if it really not necessary - I'll delete it in another PR.